client/src/components/ManageArtist/ArtistFormColors.tsx

```diff
@@ -8,17 +8,17 @@ const ArtistFormColors = () => {
   return (
     <>
       <label>
-        <strong>{t("customColors")}</strong>
+        <h2>{t("customColors")}</h2>
```

Comment:
I don't think this should be an `<h2>` but instead the `<label>` should be styled appropriately